src/main/java/com/example/fizzbuzz/controller/FizzBuzzDBController.java

```diff
@@ -0,0 +1,111 @@
+package com.example.fizzbuzz.controller;
+
+import com.example.fizzbuzz.model.ConnectFizzBuzzDB;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import java.sql.Connection;
+import java.sql.Statement;
+
+
+@RestController
+public class FizzBuzzDBController {
+
+    public static String setResult(Integer myNumber, String result){
+        if (myNumber % 3 == 0){
+            result += "Fizz";
+        }
+        if (myNumber % 5 == 0){
+            result += "Buzz";
+        }
+        return result;
+    }
+
+
+    @GetMapping("/insertValue")
+    public static void addValues(@RequestParam(value = "id") Long id, @RequestParam (value = "myNumber") Integer myNumber, @RequestParam (value = "name") String name) {
+
+        Connection connection = null;
+        Statement statement = null;
+
+        ConnectFizzBuzzDB objConnectFizzBuzzDB= new ConnectFizzBuzzDB();
+        connection = objConnectFizzBuzzDB.get_connection();
+
+
+        String result = "";
+        result = setResult(myNumber, result);
```

Comment:
This code seems to be repeated in each method, can we extract to to new method and reuse it everywhere?

Reply:
I committed new version of code on this branch